python/mdvtools/tests/mock_anndata.py

```
@@ -0,0 +1,626 @@
+            # For dense arrays, use numpy copy
+            adata.layers['counts'] = np.array(adata.X, copy=True)
+        
+        # Log-normalized data - use chunked normalization for large datasets
+        if use_chunked_layers and scipy.sparse.issparse(adata.X):
+            # Use chunked normalization directly on sparse matrix
+            log_data = chunked_log1p_normalization(adata.X)
+        else:
+            # For smaller datasets or when chunking is disabled, use traditional approach
+            if scipy.sparse.issparse(adata.X):
+                # Type: ignore because linter doesn't understand sparse matrix methods
+                X_dense = adata.X.toarray() # type: ignore
```

Flake8 (7.2.0)

[error] at least two spaces before inline comment

(E261)